### DISCUSSION TITLE
八方旅人

### DISCUSSION THREAD

**Discussion by ginhton** [Gamuxorg/bbs#224] [投递和请求游戏]
游戏名称：八方旅人
游戏名称(英文)：OCTOPATH TRAVELER
下载地址 :（如果有的话）：
steam 地址： https://store.steampowered.com/app/921570/OCTOPATH_TRAVELER/

ps: 虽然我觉得大概率弄不下来，哈哈。

**Comment by hyh19962008** [comment]
不得行，暂时没破解，放弃吧

**Comment by shouhuanxiaoji** [comment]
不是弄不弄得下的问题，这游戏只有win版，可以试试wine
